app/controllers/support_interface/application_forms_controller.rb

```diff
@@ -0,0 +1,19 @@
+module SupportInterface
+  class ApplicationFormsController < SupportInterfaceController
+    def index
+      application_forms = ApplicationForm.includes(:application_choices)
+
+      @application_forms = application_forms.map do |application_choice|
+        ApplicationFormPresenter.new(application_choice)
+      end
```

Comment:
this looks like a potential presenter `ApplicationsFormPresenter` (the plural form), not sure if it worths the work, up to you